static/css/perdiem.css

```diff
@@ -4,7 +4,7 @@ h1, h2, h3, h4, h5, h6, p, li, input [type=submit] {
 }
 
 h4 [type=submit] {
-    font-family: 'Roboto', sans-serif;
+    font-family: 'roboto', sans-serif;
```

Comment:
I noticed that on safari it wouldn't load (locally) the font unless it was in a lowercase "robot". The other strange thing I noticed was that it worked on the beta even though it didn't work on my local machine. Not really sure why that was happening.